sklearn/tests/test_calibration.py

```diff
@@ -282,9 +300,11 @@ def test_calibration_nan_imputer():
     clf = Pipeline(
         [('imputer', SimpleImputer()),
          ('rf', RandomForestClassifier(n_estimators=1))])
-    clf_c = CalibratedClassifierCV(clf, cv=2, method='isotonic')
-    clf_c.fit(X, y)
-    clf_c.predict(X)
+    for ensemble in (True, False):
```

Comment:
we probably don't need to check the different ensembles here?

Reply:
The cv in two different paths of code depending on the ensemble parameter, so I think it is good to check both.